fbgemm_gpu/codegen/embedding_backward_code_generator.py

```diff
@@ -1316,3 +1316,4 @@ def main() -> None:
 
 if __name__ == "__main__":
     main()
+    # hipify_gen()
```

Comment:
Please do not leave a meaningless comment.

Reply:
It's removed.